languages/elixir/exercises/concept/basics/.docs/instructions.md

```diff
@@ -0,0 +1,39 @@
+In this exercise you're going to write some code to help you cook a brilliant lasagna from your favorite cooking book.
+
+You have four tasks, all related to the time spent cooking the lasasgna.
+
+### 1. Define the expected oven time in minutes
+
+Define the `Lasagna.expected_minutes_in_oven/0` method that does not take any parameters and returns how many minutes the lasagna should be in the oven. According to the cooking book, the expected oven time in minutes is 40:
```

Comment:
My only remaining nit is that this notation with the arity is not explained in the introduction anywhere, so people might be a bit confused as to its meaning. Maybe one very simple mention in the introduction (and after) document would suffice.

Reply:
Yes, that's a good point. It's how functions are referred to often in elixir literature, so it would be a good idea to include it.

Reply:
Excellent!

Reply:
Added